index.js

```diff
@@ -36,6 +36,19 @@ function stack(results) {
 
 		i++;
 
+		// Don't print the full stack but the only useful line showing
+		// the actual test file stack
+		var reg = /\((?:[\\\/](?:(?!node_modules[\\\/]ava[\\\/])[^:\\\/])+)+:\d+:\d+\)/;
```

Comment:
Mind updating the regex?

```
(?:^(?! {4}at).{6})|(?:\((?:[\\\/](?:(?!node_modules[\\\/]ava[\\\/])[^:\\\/])+)+:\d+:\d+\))
```

This won't kill the error message and won't filter out lines that don't start with `at`.